armory/datasets/common.py

```diff
@@ -0,0 +1,177 @@
+from functools import lru_cache
+import json
+import hashlib
+from pathlib import Path
+
+import tensorflow_datasets as tfds
+
+from armory import paths
+from armory.logs import log
+
+SUBDIR = "new_builds"
```

Comment:
is the use of the `/armory/datasets/new_builds` subdir temporary and intended to separate out data that's been built in the past from regular armory use? If no, this might be an unexpected location to be storing future builds in the long-term

Reply:
Yes, correct.